build.gradle

```diff
@@ -23,6 +23,11 @@ buildscript {
     }
 }
 
+// Check for Java 8
+if(JavaVersion.current() != JavaVersion.VERSION_1_8){
```

Comment:
Please move this a couple of lines lower (to line 46) next to the compatibility check.